Source/NonlinearSolvers/CurlCurlMLMGPC.H

```diff
@@ -323,7 +310,12 @@ void CurlCurlMLMGPC<T,Ops>::Apply (T& a_x, const T& a_b)
         // Copy initial guess to local object
 #if defined(WARPX_DIM_1D_Z)
         // Missing dimensions is x,y in WarpX and y,z in AMReX
-        WARPX_ABORT_WITH_MESSAGE("CurlCurlMLMGPC not yet implemented for 1D");
+        Array<MultiFab,3> solution { MultiFab(*x_mfarrvec[n][2], make_alias, 0, 1),
+                                     MultiFab(*x_mfarrvec[n][1], make_alias, 0, 1),
+                                     MultiFab(*x_mfarrvec[n][0], make_alias, 0, 1) };
+        Array<MultiFab,3> rhs      { MultiFab(*b_mfarrvec[n][2], make_alias, 0, 1),
+                                     MultiFab(*b_mfarrvec[n][1], make_alias, 0, 1),
+                                     MultiFab(*b_mfarrvec[n][0], make_alias, 0, 1) };
```

Comment:
Why did you map the AMReX to WarpX dimensions as 0,1,2 -> 2,1,0 and not 0,1,2 -> 2,0,1?

Reply:
This was clarified in a meeting based on the available documentation: https://warpx.readthedocs.io/en/latest/developers/dimensionality.html.